opesci/fields.py

```diff
@@ -359,13 +167,15 @@ def set_dt(self, dt):
         """
         self.dt = dt
 
-    def set_media_param(self, media_param):
+    def associate_stress_fields(self, sfields):
         """
-        - set the media parameters beta, lambda, mu
-        - these parameters should be used for the updating kernel
-        - only needed when read data from file
+        link this velocity field to a list of stress field
+        e.g. Vx will be associated with Txx, Txy, Txz
+        link compression stress field with other compression stress field
+        e.g. Txx with Txx, Tyy, Tzz
+        :param sfields: list of associated stress field
         """
-        self.media_param = media_param
+        self.sfields = sfields
```

Comment:
stress field also need to be associated with another stress field for bc.
e.g. Txx need Txz to update at the z boundary